### PR TITLE
Enum heades

### PR DESCRIPTION
**Pull request by GustavoZHA**

### CODE REVIEW

File: .env.develop [hunk 1]
```diff
@@ -1,37 +1,12 @@
...
-CLIENT_ID_KEY="client_id"
 CLIENT_ID_VALUE="3MVG9cHH2bfKACZZFzqzsZvLw_bwYyAsG1d.ZZ3uLi1mDLktOy0e_T313M4YLQtiV6_2CTKaUNdQq2KWNvZEy"
-CLIENT_SECRET_KEY="client_secret"
 CLIENT_SECRET_VALUE="A5F85270B193B5E7C6BBBD7E4240B5C19E2C1250DD76D89A09B023A024235308"
-USERNAME_KEY="username"
 USERNAME_VALUE="gzhuanca@freeorg01.com"
-PASSWORD_KEY="password"
 PASSWORD_VALUE="4219508Salesforce"
 TOKEN_SECURITY="lq73zO0TLHsIo31qx0oz7ich"
+
```
Comment (jpgonzales27) [inline .env.develop]: This line should be deleted

File: .env.develop [hunk 2]
```diff
@@ -1,37 +1,12 @@
...
-OPPORTUNITY="sobjects/Opportunity"
-OPPORTUNITY_TO_INTERACT="sobjects/Opportunity/{opportunityId}"
-OPPORTUNITY_ID="opportunityId"
-CAMPAIGN_ID="campaignId"
-CAMPAIGN="/sobjects/Campaign/"
-CAMPAIGN_TO_INTERACT="sobjects/Campaign/{campaignId}"
-INDIVIDUAL_ID="individualId"
-INDIVIDUAL="sobjects/Individual"
-INDIVIDUAL_TO_INTERACT="sobjects/Individual/{individualId}"
+
```
Comment (jpgonzales27) [inline .env.develop]: this line too

File: src/main/java/generalsetting/EndPoint.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    URL_BASE(dotenv.get("URL_BASE"),  "",""),
+    PRODUCT("sobjects/Product2/",  "sobjects/Product2/{productId}","productId"),
+    GROUP("chatter/groups",       "chatter/groups/{groupId}",   "groupId"),
+    ACCOUNT("sobjects/Account",  "sobjects/Account/{accountId}","accountId"),
+    CONTACT("sobjects/Contact",  "sobjects/Contact/{contactId}","contactId"),
+    OPPORTUNITY("sobjects/Opportunity",  "sobjects/Opportunity/{opportunityId}","opportunityId"),
+    CAMPAIGN("/sobjects/Campaign/",  "sobjects/Campaign/{campaignId}","campaignId"),
+    INDIVIDUAL("sobjects/Individual",  "sobjects/Individual/{individualId}","individualId");
+
+
```
Comment (jpgonzales27) [inline src/main/java/generalsetting/EndPoint.java]: this line too

File: .env.develop (hunk 1 above)
Comment (GustavoZHA) [reply .env.develop]: ok

**Pull request merged**